test/general/streaming.js

```diff
@@ -874,7 +874,7 @@ function tests() {
     });
     expect(openpgp.stream.isStream(signed)).to.equal(expectedType);
     const reader = openpgp.stream.getReader(signed);
-    expect((await reader.readBytes(31)).toString('utf8')).to.equal('-----BEGIN PGP SIGNATURE-----\r\n');
+    expect((await reader.readBytes(31)).toString('utf8')).to.equal('-----BEGIN PGP SIGNATURE-----\n\n');
     dataArrived();
     reader.releaseLock();
     await openpgp.stream.cancel(signed, new Error('canceled by test'));
```

Comment:
This test fails in node because the stream does not get cancelled. How is this related to the changes?

Reply:
This is a bit of a guess, but you could try reading one fewer byte, and then expect one fewer newline? As the second newline you're currently expecting comes from a different chunk than the previous expected value.